uaclient/exceptions.py

```diff
@@ -306,6 +306,18 @@ def __str__(self):
         )
 
 
+class EntitlementNotEnabledError(UserFacingError):
+    """An exception raised when enabling of an entitlement fails"""
+
+    pass
+
+
+class FullAutoAttachFailureError(UserFacingError):
```

Comment:
We could already create the exception with the message we want here directly, like we do on `APTInstallError` for example. In that way, we can move the message into the `message` module and just use it here